sda/cmd/finalize/finalize.go

```diff
@@ -144,24 +143,24 @@ func main() {
 			completeMsg, _ := json.Marshal(&c)
 			err = schema.ValidateJSON(fmt.Sprintf("%s/ingestion-completion.json", conf.Broker.SchemasPath), completeMsg)
 			if err != nil {
-				log.Errorf("Validation of outgoing message failed, reason: (%v)", err)
+				log.Errorf("Validation of outgoing message ingestion-completion failed, reason: (%v). Message body: %s\n", err, string(completeMsg))
 
 				continue
 			}
 
 			accessionIDExists, err := db.CheckAccessionIDExists(message.AccessionID, fileID)
 			if err != nil {
-				log.Errorf("CheckAccessionIdExists failed, reason: %v ", err)
+				log.Errorf("CheckAccessionIdExists failed, file-id: %s, reason: %v ", fileID, err)
 				if err := delivered.Nack(false, true); err != nil {
-					log.Errorf("failed to Nack message, reason: (%v)", err)
+					log.Errorf("failed to Nack message, reason: %v", err)
 				}
 
 				continue
 			}
 
 			switch accessionIDExists {
 			case "duplicate":
-				log.Debugf("Seems accession ID already exists (corr-id: %s, accessionid: %s", delivered.CorrelationId, message.AccessionID)
+				log.Errorf("Accession ID already exists in the system. (accession-id: %s)\n", message.AccessionID)
```

Comment:
```suggestion
				log.Errorf("accession ID already exists in the system. (accession-id: %s)\n", message.AccessionID)
```

Reply:
We probably need to do a refactoring. Error log with capitalized sentences exist at many places in the code base.